org.uqbar.project.wollok.lib/src/wollok/lib.wlk

```diff
@@ -0,0 +1,130 @@
+/** 
+ * Console is a global wollok object that implements a character-based console device
+ * called "standard input/output" stream 
+ */
+object console {
+
+	/** Prints a String with end-of-line character */
+	method println(obj) native
+	
+	/** Reads a line from input stream */
+	method readLine() native
+	
+	/** Reads an int character from input stream */
+	method readInt() native
+}
+
+/**
+ * Assert object simplifies testing conditions
+ */
+object assert {
+
+	/** 
+	 * Tests whether value is true. Otherwise throws an exception.
+	 * Example:
+	 * 		var number = 7
+	 *		assert.that(number.even())   ==> throws an exception "Value was not true"
+	 * 		var anotherNumber = 8
+	 *		assert.that(anotherNumber.even())   ==> no effect, ok		
+	 */
+	method that(value) native
+	
+	/** Tests whether value is false. Otherwise throws an exception. 
+	 * @see assert#that(value) 
+	 */
+	method notThat(value) native
+	
+	/** 
+	 * Tests whether two values are equal, based on wollok == method
+	 * 
+	 * Example:
+	 *		 assert.equals(10, 100.div(10)) ==> no effect, ok
+	 *		 assert.equals(10.0, 100.div(10)) ==> no effect, ok
+	 *		 assert.equals(10.01, 100.div(10)) ==> throws an exception 
+	 */
+	method equals(expected, actual) native
+	
+	/** Tests whether two values are equal, based on wollok != method */
+	method notEquals(expected, actual) native
+	
+	/** 
+	 * Tests whether a block throws an exception. Otherwise an exception is thrown.
+	 *
+	 * Example:
+	 * 		assert.throwsException({ 7 / 0 })  ==> Division by zero error, it is expected, ok
+	 *		assert.throwsException("hola".length() ) ==> throws an exception "Block should have failed"
+	 */
+	method throwsException(block) native
+	
+	/**
+	 * Throws an exception with a custom message. Useful when you reach an unwanted code in a test.
+	 */
+	method fail(message) native
+	
+}
+
+class StringPrinter {
+	var buffer = ""
+	method println(obj) {
+		buffer += obj.toString() + "\n"
+	}
+	method getBuffer() = buffer
+}	
+
+object wgame {
```

Comment:
I don't know, I haven changed the distribution in packages because everything will destroy. I don't know what do you think @Juancete @PalumboN and the others?